src/strtree.c

```diff
@@ -129,6 +196,54 @@ void query_callback(void *item, void *user_data)
     kv_push(npy_intp, *(npy_intp_vec *)user_data, (npy_intp) item);
 }
 
+/* Evaluate the predicate function against a prepared version of geom
+ * for each geometry in the tree specified by indexes in target_indexes.
+ * out_indexes is updated in place with the indexes of the geometries in the
+ * tree that meet the predicate.
+ * Returns the number of geometries that met the predicate.
```

Comment:
Can you more explicitly document here the different arguments? (it's not that easy to grasp I find)